src/style.css

```diff
@@ -0,0 +1,46 @@
+body,
+div {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url(images\star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:first-of-type) {
+  margin-left: 4px;
+}
+
+.stars--0 :nth-child(-n + 0) {
+  background-image: url(images\star-active.svg);
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(images\star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(images\star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(images\star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(images\star-active.svg);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(images\star-active.svg);
+}
```

Comment:
combine these selectors. There is a suggestion in the checklist

Reply:
Done!